ballerina/constants.bal

```diff
@@ -76,3 +76,6 @@ const GRAPHQL_WS = "graphql-ws";
 const GRAPHQL_TRANSPORT_WS = "graphql-transport-ws";
 const WS_SUB_PROTOCOL = "Sec-WebSocket-Protocol";
 const DEFAULT_VALUE = "default";
+
+// Error messages
+const UNABLE_TO_PERFORM_DATA_BINDING = "Unable to perform data binding";
```

Comment:
IMO it's better to define this in a `common_utils` function since this is an error message.

Reply:
+1, moved